src/Features/CSharp/Portable/MakeMethodAsynchronous/CSharpMakeMethodAsynchronousCodeFixProvider.cs

```diff
@@ -35,7 +35,9 @@ protected override string GetMakeAsyncVoidFunctionResource()
 
         protected override bool IsMethodOrAnonymousFunction(SyntaxNode node)
         {
-            return node.IsKind(SyntaxKind.MethodDeclaration) || node.IsAnyLambdaOrAnonymousMethod();
+            return node.IsKind(SyntaxKind.MethodDeclaration)
+                || node.IsAnyLambdaOrAnonymousMethod()
+                || node.IsKind(SyntaxKind.LocalFunctionStatement);
```

Comment:
duplicated code.  can we move to an extension?  (i actually think we have one that already does this somewhere).